assets/pages/oneoff-contributions/components/nameFormField.jsx

```diff
@@ -44,12 +60,10 @@ function mapStateToProps(state) {
 
 }
 
-function mapDispatchToProps(dispatch: Dispatch<Action>) {
+function mapDispatchToProps(dispatch: Dispatch<*>) {
```

Comment:
I may have missed something here, but is this no longer able to be parameterised with `Action`?